content/articles/domains-day.markdown

```diff
@@ -0,0 +1,24 @@
+---
+title: .DAY Domains
+excerpt: This article explains the requirements and special procedures for .DAY domain names.
+categories:
+- Domains
+---
+
+# .DAY Domain Names
+
+* TOC
+{:toc}
+
+---
+
+This article explains the requirements and special procedures for .DAY domain names.
+
+
+## Registering a .DAY domain {#registering}
+
+.day is a secured namespace, meaning that websites using a .day domain require an SSL certificate to work; .day websites can only be accessed by web browsers using HTTPS through an encrypted and secured connection.
+
+You can obtain a certificate for your .day domain through DNSimple. To learn more, visit [SSL Certificates](/articles/ssl-certificates).
```

Comment:
What about a call to action to the site.

```suggestion
You can [obtain a certificate for your .day domain](https://dnsimple.com/ssl-certificates) through DNSimple. To learn more, visit [SSL Certificates](/articles/ssl-certificates).
```

Reply:
Done in [4217023](https://github.com/dnsimple/dnsimple-support/pull/949/commits/42170232600f7dd4f3055c089b35c01e8fd1ffc4)